### ISSUE TITLE
Usage of the **upstream** keyword in Episode 3 can cause confusion 

### ISSUE THREAD

**Issue by nturaga** [LibraryCarpentry/lc-git#77]
## In Episode 3:  Pushing Changes

Do not use the word **upstream** to represent the transmission of the `.git` metadata the remote repository. 
 
The flag `-u` hasn’t been explained in the context of setting a **upstream branch** or tracking branch. The usage of the word ‘upstream’ would make more sense if the `-u` is explained along with it’s long form flag `--set-upstream`.

Another reason, to avoid using this term is because the name `upstream` is sometimes used to track remotes other than the `origin`. 

The excerpt from the episode 3 that I refer to is copied below:

```
When we do a git push, we will see Git ‘pushing’ changes upstream to GitHub. 
Because our file is very small, this won’t take long but if we had made a lot of 
changes or were adding a very large repository, we might have to wait a little 
longer. We can check where we’re at with git status.
```

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

If this issue is about a specific episode within a lesson, please provide its link or filename.

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  

---

**Comment by dcmcand** [comment]
Hi @nturaga,
Do you think just removing the work upstream so that it reads "When we do a git push, we will see Git ‘pushing’ changes upstream to GitHub." would remove the confusion, or do  you think we need to explain the term upstream which is used pretty frequently in git?

**Comment by nturaga** [comment]
I was hoping we could avoid the use of `upstream` until it is explained why that term is coming up. Explaining seems the best way forward. 

That sentence works without the upstream as well ` "When we do a git push, we will see Git ‘pushing’ changes to GitHub."`.

**Comment by dcmcand** [comment]
Sounds good. Are you comfortable sending a pull request?

**Comment by dcmcand** [comment]
Closed by #86
